### PR TITLE
Fix the bug for MNX rxn ID in notes field

### PR DESCRIPTION
**Pull request by feiranl**
### Main improvements in this PR:
* Fix the bug for MNX rxn IDs those exist in the rxnNotes field #225 
* Fix the dependency for `sol.obj`, since it is not generated from `optimizeCbModel` in the new version of cobra toolbox

**I hereby confirm that I have:**
- [x] Tested my code with [all requirements](https://github.com/SysBioChalmers/yeast-GEM#required-software---contributor) for running the model
- [x] Selected `devel` as a target branch (top left drop-down menu)
- [x] If needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/yeast-GEM) about this PR

### CODE REVIEW

File: ModelFiles/dependencies.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MATLAB	9.7.0.1190202 (R2019b)
 libSBML	5.17.0
-RAVEN_toolbox	commit 1300c01
+RAVEN_toolbox	commit 6a77071
```
Comment (BenjaSanchez) [inline ModelFiles/dependencies.txt:3]: please switch RAVEN back to master, as the latest version `2.4.0` is now compliant with BiGG ids. However cobratoolbox is still not fixed in master, so keep it in development :)

File: ModelFiles/xml/yeastGEM.xml [hunk 2]
```diff
@@ -199653,12 +199653,17 @@
         <notes>
           <body xmlns="http://www.w3.org/1999/xhtml">
             <p>Confidence Level: 0</p>
-            <p>NOTES: added after Gapfilling (PR #185); MNXR94845</p>
+            <p>NOTES: added after Gapfilling (PR #185); seed:rxn09802; unknown transporter</p>
```
Comment (BenjaSanchez) [inline ModelFiles/xml/yeastGEM.xml:199656]: SEED ids are also able to be stored in annotation, use the `rxnSEEDID` field for consistency (notes should only store what cannot go in annotation).

File: ModelFiles/xml/yeastGEM.xml [hunk 3]
```diff
@@ -199680,12 +199685,17 @@
         <notes>
           <body xmlns="http://www.w3.org/1999/xhtml">
             <p>Confidence Level: 0</p>
-            <p>NOTES: added after Gapfilling (PR #185); MNXR95431</p>
+            <p>NOTES: added after Gapfilling (PR #185); rhea:27814; unknown transporter</p>
```
Comment (BenjaSanchez) [inline ModelFiles/xml/yeastGEM.xml:199688]: same with RHEA: use `rxnRheaID` to store them

File: ModelFiles/xml/yeastGEM.xml [hunk 4]
```diff
@@ -199809,12 +199839,17 @@
         <notes>
           <body xmlns="http://www.w3.org/1999/xhtml">
             <p>Confidence Level: 0</p>
-            <p>NOTES: added after Gapfilling (PR #185); MNXR102871</p>
+            <p>NOTES: added after Gapfilling (PR #185); rhea:32823; bigg:PItg; unknown transporter</p>
```
Comment (BenjaSanchez) [inline ModelFiles/xml/yeastGEM.xml:199842]: there are some BiGG ids that can be stored in `rxnBiGGID`

File: ModelFiles/xml/yeastGEM.xml (hunk 2 above)
Comment (feiranl) [reply ModelFiles/xml/yeastGEM.xml:199656]: If we are going to include seed IDs in the model, should we include them all for all possible reactions?

Comment (feiranl) [reply ModelFiles/xml/yeastGEM.xml:199656]: I have opened a new issue #231  to add rhea and modelseed IDs in a systematic way. So here, we don't address this.

File: ModelFiles/xml/yeastGEM.xml (hunk 3 above)
Comment (BenjaSanchez) [reply ModelFiles/xml/yeastGEM.xml:199688]: will be handled in #231

File: ModelFiles/xml/yeastGEM.xml (hunk 4 above)
Comment (BenjaSanchez) [reply ModelFiles/xml/yeastGEM.xml:199842]: mostly addressed in [c82e696](https://github.com/SysBioChalmers/yeast-GEM/pull/226/commits/c82e6962ec020ec14a231bf3de36286c3d1a79d5) (a few are still left but they can be included when moving all notes info to annotation as mentioned in #231)

**Pull request merged**